draft-ietf-rats-corim.md

```diff
@@ -1932,12 +1941,13 @@ This can be acheived by sorting the triples before processing, by repeating proc
 
 Reference Value Providers (RVP) publish Reference Values triples that are matched against ACS entries.
 Reference Values may describe multiple acceptable states for Attesters; hence "matching" determines that Evidence (contained in the ACS) satisfies an appropriate subset of the available Reference Values.
+
 If the appropriate subset matches, the authority of the RVP is added to the appropriate ACS entries.
 
-The Verifier compares each `reference-triple-record` against ACS entries as described in {{sec-match-one-se}}, where the `reference-triple-record` takes the place of a `stateful-environment-record`.
-If all fields of the `reference-triple-record` match the ACS, then the Verifier MUST add the RVP authority to each matching ACS field.
+The Verifier compares each `rv` reference value condition ECT against ECT entries in the ACS as described in {{sec-match-one-se}}.
```

Comment:
The referenced section is not properly named for what this is needed to match all mkey/mval pairs

"Processing a stateful environment against the Appraisal Claims Set {#sec-match-one-se}"

Reply:
The corroboration section removed the reference to sec-match-one-se for now.
The wording now says "... ECTs are identical...". There should be a section that describes how two ECTs are compared. Once that section is complete, a reference to it can be added here.